contrib/format-excel/pom.xml

```diff
@@ -64,6 +64,11 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.pjfanning</groupId>
+      <artifactId>excel-streaming-reader</artifactId>
+      <version>2.3.1</version>
```

Comment:
Please use the latest version of the reader: https://mvnrepository.com/artifact/com.github.pjfanning/excel-streaming-reader